server/src-lib/Hasura/RQL/DDL/ComputedField.hs

```diff
@@ -76,6 +78,11 @@ data InvalidTableArgument
   | ITANotTable !QualifiedTable !FunctionTableArgument
   deriving (Show, Eq)
 
+data InvalidSessionArgument
+  = ISANotFound !FunctionArgName
+  | ISANotJSON !FunctionSessionArgument
+  deriving (Show, Eq)
```

Comment:
Hi, it'd be nice to see the tests which fails with these new validation errors. If added, please ignore this comment and feel free to mark it as resolved.

Reply:
Thanks, Rakesh - I have added two tests corresponding to the two errors.